backend/package.json

```diff
@@ -16,7 +16,7 @@
     "lint-staged": "lint-staged"
   },
   "dependencies": {
-    "@relay-protocol/abis": "0.0.5",
```

Comment:
I need to revert this before merging (and I would deploy a new version of the package!)